data/hlint.yaml

```diff
@@ -835,6 +835,14 @@
     - error: {lhs: "Control.Lens.nullOf (Control.Lens.at a)", rhs: "False"}
     - error: {lhs: "Control.Lens.nullOf (a . Control.Lens.at b)", rhs: "Control.Lens.nullOf a"}
 
+- group:
+    name: template-haskell
```

Comment:
Do we need a separate group? Why not just fold them into the main set?

Reply:
Good question :) Used a hint name instead!